apps/web/components/CommunityCard.tsx

```diff
@@ -1,236 +1,43 @@
+import { RectangleGroupIcon } from "@heroicons/react/24/outline";
 
-export type StakesMemberType =
-  isMemberQuery["members"][number]["memberCommunity"];
-
-type CommunityQuery = NonNullable<
-  NonNullable<getCommunitiesByGardenQuery["tokenGarden"]>["communities"]
->[number];
-
-type CommunityCardProps = CommunityQuery & {
-  tokenGarden: TokenGarden | undefined;
-} & {
-  covenantData?: { logo: string; covenant: string };
+type CommunityCardProps = {
+  name: string;
+  members: number;
+  pools: number;
+  id: string;
 };
 
 export function CommunityCard({
-  covenantData,
-  communityName: name,
-  id: communityAddress,
-  strategies,
+  name,
   members,
-  registerStakeAmount,
-  protocolFee,
-  communityFee,
-  tokenGarden,
+  pools,
+  id,
 }: CommunityCardProps) {
-  const { address: accountAddress } = useAccount();
-  const [memberStakedTokens, setMemberStakedTokens] = useState<string>("0");
   const pathname = usePathname();
-
-  const urqlClient = useUrqlClient();
-
-  const chainId = getChainIdFromPath();
-
-  const runIsMemberQuery = useCallback(async () => {
-    if (accountAddress === undefined) {
-      return;
-    }
-    const { data: result, error } = await queryByChain<isMemberQuery>(
-      urqlClient,
-      chainId,
-      isMemberDocument,
-      {
-        me: accountAddress.toLowerCase(),
-        comm: communityAddress.toLowerCase(),
-      },
-    );
-
-    if (result && result.members.length > 0) {
-      const stakedTokens =
-        result.members?.[0]?.memberCommunity?.[0]?.stakedTokens;
-      const memberStakedTokens =
-        typeof stakedTokens === "string" ? stakedTokens : "0";
-
-      setMemberStakedTokens(memberStakedTokens);
-    }
-  }, [accountAddress]);
-
-  useEffect(() => {
-    runIsMemberQuery();
-  }, [accountAddress]);
-
-  const pools = strategies ?? [];
-
-  members = members ?? [];
-  let registerToken = tokenGarden?.id ?? "0x0";
-  registerStakeAmount =
-    registerStakeAmount !== undefined ? BigInt(registerStakeAmount) : 0n;
-
-  const signalingPools = pools.filter(
-    (pool) => pool.config?.proposalType === "0" && pool.isEnabled,
-  );
-
-  const fundingPools = pools.filter(
-    (pool) => pool.config?.proposalType === "1" && pool.isEnabled,
-  );
-
-  const poolsInReview = pools.filter((pool) => !pool.isEnabled);
-
   return (
-    <>
-      <div className="border2 rounded-lg bg-white p-4">
-        <div className="flex w-full flex-col items-center justify-center gap-10 rounded-xl bg-base-100 py-4">
-          <h2 className="text-center font-press text-3xl text-info-content">
-            {name}
-          </h2>
-          <CommunityProfile
-            communityAddress={communityAddress as Address}
-            name={name as string}
-            covenantData={covenantData}
-          />
+    <Card href={`${pathname}/${id}`} className="w-[273px]">
+      <Image
+        src={commImg}
+        alt={`${name} community`}
+        className="mb-2 h-[100px] bg-slate-200"
```

Comment:
suggestion: remove classNames here ?
because adding a bg makes a weird background on the image

Reply:
Left it there because this image is just a placeholder, but i remove it now if it is confusing